security/security.c

```diff
@@ -687,6 +737,16 @@ static void __init lsm_early_task(struct task_struct *task)
 #include <linux/lsm_hook_defs.h>
 #undef LSM_HOOK
 
+
+#define TRY_TO_STATIC_CALL_INT(NUM, R, HOOK, ...)		\
+	if (static_branch_unlikely(&HOOK_STATIC_CHECK(HOOK, NUM))) {		\
+		R = static_call(HOOK_STATIC_CALL(HOOK, NUM))(__VA_ARGS__); 	\
+		if (R != 0)					\
+			break;					\
+	}
```

Comment:
If we were willing to get deep into the static_call implementation, I think it might be possible to avoid the static key. Let's discuss f2f.